### ISSUE TITLE
How to Finetune object detection part for 3D tracking?

### ISSUE THREAD

**Issue by metacav** [xingyizhou/CenterTrack#281]
Can I Fine-tune CenterNet with 2D image with bounding boxes to change the labels for 3D tracking? Any suggestions are sincerely appreciated.
